src/QueryReflection/PdoQueryReflector.php

```diff
@@ -71,6 +71,14 @@ protected function simulateQuery(string $queryString)
                 throw new ShouldNotHappenException('Failed to get column meta for column index '.$columnIndex);
             }
 
+            /*
+             * Native type may not bo set, for example in case of JSON column.
+             * @phpstan-ignore-next-line
+             */
+            if (!isset($columnMeta['native_type'])) {
+                $columnMeta['native_type'] = \PDO::PARAM_INT === $columnMeta['pdo_type'] ? 'INT' : 'STRING';
```

Comment:
According to pdo_mysql src, `native_type` can be missing from the return value:
https://github.com/php/php-src/blob/14319c203cb13a7cc0644fb4276d6e62338fda5e/ext/pdo_mysql/mysql_statement.c#L806-L808

and in that case, we may fallback to the `pdo_type`, problem is, according to my understanding of the sources, this can be either INT or STR... https://github.com/php/php-src/blob/14319c203cb13a7cc0644fb4276d6e62338fda5e/ext/pdo_mysql/mysql_statement.c#L811-L827.